app/views/govuk_publishing_components/components/docs/layout_for_public.yml

```diff
@@ -27,6 +27,12 @@ examples:
     description: This allows the header to be omitted which is currently used when rendering CSV previews from Whitehall
     data:
       omit_header: true
+  blue_bar_background:
+    description: For use when a page has a heading component with a background colour.
+    data:
+      full_width: true
+      blue_bar: true
+      blue_bar_background: "explore"
```

Comment:
There does not appear to be an input for `blue_bar_background`, should this be `blue_bar_background_colour`?

If so, does the string also need to be changed from "explore" to "browse" so the [grey background color](https://github.com/alphagov/govuk_publishing_components/pull/3380/files#diff-841f2eaa57914d8e9aabb60cae32faa0fb1d09fcfd485e89e60f4530d73d1f92R12) is applied?